src/components/TableComponent/__snapshots__/TableComponent.test.jsx.snap

```diff
@@ -1150,6 +1150,7 @@ ShallowWrapper {
       },
     },
   },
+  Symbol(enzyme.__childContext__): null,
```

Comment:
I am not sure why these snapshots are in here.  Looking into it before merging PR.

Reply:
If you pulled in enzyme 3.9.0? That might explain it.